assets/js/components.js

```diff
@@ -57,6 +57,19 @@ function accordion($el) {
   return new Accordion($el);
 }
 
+function togglePassword($el) {
+  var fieldSelector =  '#' + $el.attr('aria-controls'),
```

Comment:
Since we're not using `#` as the selector, this does not seem to be functioning correctly. Should we change the actual selector to match up accordingly?